contributors/devel/development.md

```diff
@@ -197,9 +203,10 @@ If everything checks out you will have an executable in the `_output/bin` direct
 
 *Note:* If you are using `CDPATH`, you must either start it with a leading colon, or unset the variable. The make rules and scripts to build require the current directory to come first on the CD search path in order to properly navigate between directories.
 
+To build everything:
 ```sh
 cd $working_dir/kubernetes
-make
+make all
```

Comment:
make is equivalent to make all...
suggest saying something to the effect of make or make all..?

Reply:
Forgot the comment. Thanks for the reminder.